Libraries/acdriver/Headers/acdriver/Driver.h

```diff
@@ -14,6 +14,7 @@
 #include <vector>
```

Comment:
I really like this change!  LGTM!

Can you explain what's stopping you from adding the process context everywhere (like, is it just too messy to pipe the context through to pbxbuild, or is it unnecessary)?